web/src/components/nomination/TopicsInput.vue

```diff
@@ -5,7 +5,8 @@
         ref="topics"
         :label="$t('nominateSpeaker.topics')"
         :items="topics"
-        item-text="en"
+        :item-text="$i18n.locale === 'ja' ? 'name_ja' : 'name_en'"
```

Comment:
One thought, but either is fine.
```suggestion
        :item-text="`name_${$i18n.local}`"
```